### ISSUE TITLE
reconnection in fight does'nt work

### ISSUE THREAD

**Issue by melharfi** [w0dm4n/Symbioz#2]
thanks for sharing this
but that feature is not working i tryed it on a different fight type.

**Comment by w0dm4n** [comment]
Hi, the feature has been disabled but you can uncomment the code to re activate it, look at the disconnection and in the approach handler

**Comment by melharfi** [comment]
Yes i found the 3 places and in fact it worked ONLY one time and when i retry the client freez in 50% loading with an error message : 
TypeError: Error #1009: Cannot access a property or method of a null object reference. 

it mean that it's not well handled and useless in the end, but at least there's some starting point ^^
thanks for the hint

**Comment by w0dm4n** [comment]
Nope it's working well with less then 100 players you certainly missed smth or i've deleted a part of it
